site/Makefile

```diff
@@ -0,0 +1,79 @@
+# Credit to the Kubernetes Website team. (https://github.com/kubernetes/website/blob/main/Makefile)
+HUGO_VERSION      = $(shell grep ^HUGO_VERSION netlify.toml | tail -n 1 | cut -d '=' -f 2 | tr -d " \"\n")
+NODE_BIN          = node_modules/.bin
+NETLIFY_FUNC      = $(NODE_BIN)/netlify-lambda
+
+# The CONTAINER_ENGINE variable is used for specifying the container engine. By default 'docker' is used
+# but this can be overridden when calling make, e.g.
+# CONTAINER_ENGINE=podman make container-image
+CONTAINER_ENGINE ?= docker
+IMAGE_REGISTRY ?= gcr.io/k8s-staging-sig-docs
```

Comment:
We don't need this bit. If I recall correctly, k/website pushes a dev image to this registry so that people don't need to build it each time locally, but we don't currently do that. So we can probably change this for `localhost:5000` or `registry.local:5000` to prevent an accidental push.

Reply:
Replaced the registry value with `registry.local:5000` until we begin to push the Kustomize docs site image.